src/commands/dev/index.ts

```diff
@@ -629,8 +648,16 @@ $ architect dev -e new_env_name_here .`));
     }
 
     const interfaces_map = DeployUtils.getInterfacesMap(flags.interface);
+
+    let env_secrets: SecretsDict = {};
+    if (flags['secrets-env']) {
+      const account = await AccountUtils.getAccount(this.app, flags.account, { ask_local_account: false });
+      env_secrets = await this.getEnvironmentSecrets(account, flags['secrets-env']);
+      flags.account = account.name;
```

Comment:
What is this used for?

Reply:
It's not really used for anything so I will remove it.